common/src/main/java/com/wynntils/screens/maps/AbstractMapScreen.java

```diff
@@ -43,8 +43,8 @@ public abstract class AbstractMapScreen extends WynntilsScreen {
     private static final float BORDER_OFFSET = 6;
 
     // Zoom is the scaling of the map. The bigger the zoom, the more detailed the map becomes.
-    private static final float MIN_ZOOM = 0.1f;
-    private static final float MAX_ZOOM = 3f;
+    public static final float MIN_ZOOM = 0.1f;
+    public static final float MAX_ZOOM = 3f;
```

Comment:
I would rather you just created these constants in the overlay class. Sharing this looks weird.